### ISSUE TITLE
Charge centres functionality not available

### ISSUE THREAD

**Issue by Academicma** [rosalindfranklininstitute/okapi-em#5]
Dear developers, when I open the Okapi plugin in napari, I get the following message on the terminal:

_Failed to import python modules needed for predicting charge centres. This functionality will be unavailable._

I am running napari on macOS 12.2 with python 3.10. Any idea?

**Comment by perdigao1** [comment]
yes. That is the package chafer. It has an engine to automatically segment charge centres using pytorch (big package) but it is under development. Just ignore it.
